src/devices/bosch.ts

```diff
@@ -576,6 +694,50 @@ const fzLocal = {
 };
 
 const definitions: Definition[] = [
+    {
+        zigbeeModel: ['RBSH-OS-ZB-EU'],
+        model: 'BSIR-EZ',
+        vendor: 'Bosch',
+        description: 'Outdoor siren',
+        fromZigbee: [fz.ias_alarm_only_alarm_1, fzLocal.ias_siren, fz.battery, fz.power_source],
+        toZigbee: [tzLocal.rbshoszbeu, tz.warning],
+        meta: {battery: {voltageToPercentage: {min: 2500, max: 4200}}},
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg', 'ssIasZone', 'ssIasWd', 'genBasic']);
+            await reporting.batteryVoltage(endpoint);
+            await endpoint.read(0x0502, [0xa000, 0xa001, 0xa002, 0xa003, 0xa004, 0xa005], boschManufacturer);
+            device.defaultSendRequestWhen = 'immediate';
+            device.save();
+            await endpoint.unbind('genPollCtrl', coordinatorEndpoint);
+        },
+        exposes: [
+            e.enum('alarm_on', ea.ALL, Object.keys(outdoorSirenState)).withDescription('Alarm turn ON/OFF'),
+            e.numeric('light_delay', ea.ALL).withValueMin(0).withValueMax(30).withValueStep(1)
+                .withUnit('s').withDescription('Flashing light delay [s]'),
+            e.numeric('siren_delay', ea.ALL).withValueMin(0).withValueMax(30).withValueStep(1)
+                .withUnit('s').withDescription('Siren alarm delay [s]'),
+            e.numeric('siren_duration', ea.ALL).withValueMin(1).withValueMax(15).withValueStep(1)
+                .withUnit('m').withDescription('Duration of the alarm siren [m]'),
+            e.numeric('light_duration', ea.ALL).withValueMin(1).withValueMax(15).withValueStep(1)
+                .withUnit('m').withDescription('Duration of the alarm light [m]'),
```

Comment:
No need to add the unit between brackets here, it is already covered in `withUnit`